provider/fs/Cargo.toml

```diff
@@ -55,11 +55,16 @@ static_assertions = { version = "1.1", optional = true }
 icu_benchmark_macros = { version = "0.3", path = "../../tools/benchmark/macros" }
 icu_locid_macros = { version = "0.3", path = "../../components/locid/macros" }
 icu_plurals = { version = "0.3", path = "../../components/plurals" }
+serde-json-core = { version = "0.4", features = ["std"] }
```

Comment:
By migrating the manifest file to serde-json-core, which I wanted to do anyway, I can make serde_json an optional dependency.

Reply:
praise: nice!